pilot/pkg/proxy/envoy/v2/eds.go

```diff
@@ -65,6 +66,12 @@ var (
 	connectionNumber = int64(0)
 )
 
+const (
+	// The range of LoadBalancingWeight is [1, 128]
```

Comment:
wasn't this fixed in envoy?

Reply:
not yet, see the proto definition

Reply:
https://github.com/envoyproxy/envoy/pull/4894#discussion_r229053992